src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -45,21 +46,31 @@ public static class TraceProvider
 			public const int LauchedStart = 1;
 			public const int LauchedStop = 2;
 		}
-		
+
 		public static Application Current { get; private set; }
 
 		public DebugSettings DebugSettings { get; } = new DebugSettings();
 
 		public ApplicationTheme RequestedTheme
 		{
-			get => _requestedTheme ?? (_requestedTheme = GetDefaultSystemTheme()).Value;
+			get
+			{
+				if (_requestedTheme != null)
+				{
+					_requestedTheme = GetDefaultSystemTheme();
+					ObserveSystemThemeChanges();
```

Comment:
Is this the right place? I think we want to observe system theme changes unconditionally, and it also seems like this would add a new listener every time `RequestedTheme` is accessed.

Reply:
The `_requestedTheme` will be `null` only initially, and after that the `if` block will never be executed again. If the user sets the theme explicitly, we no longer need to observe system theme changes, as they will never be applied (once developer sets the `RequestedTheme` once, it is forever in their hands, and will never again react to system)

Reply:
Ok, that makes sense.

I think the condition is wrong then - shouldn't it be `if (_requestedTheme == null)`?